packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -84,6 +84,19 @@ export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
           },
           '&:focus': {
             color: semanticColors.link
+          },
+          '&:focus::after': {
```

Comment:
When the link is focused, it will show the focus rectangle regardless of the root class `ms-Fabric--isFocusVisible` (I think).